backend/src/test/java/neuefische/capstone/bpmndiagram/BpmnDiagramIntegrationTest.java

```diff
@@ -0,0 +1,204 @@
+package neuefische.capstone.bpmndiagram;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+class BpmnDiagramIntegrationTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Autowired
+    private ObjectMapper objectMapper;
```

Comment:
Den ObjectMapper zu injecten ist ok - besser finde ich aber, wenn Du ihn hier im Test mit `new` erzeugst. Dadurch wird der test-code in diesem Punkt unabhängig vom production code.
Ungewöhnliche / ungewollte Konfigurationen des ObjectMappers fallen dann sofort auf.